main.cpp

```diff
@@ -122,6 +125,18 @@ class CDnsSeedOpts {
           break;
         }
 
+        case 's': {
+          if (!strcmp(optarg, "NODE_NETWORK"))
+            requiredNodeFlags = NODE_NETWORK;
```

Comment:
Yes. Good idea.
But how would you parse the `optarg`?
We could use a one char flag identifier (B for BLOOM, W for WITNESS) and use a optarg of `BW`?